src/GrassPokemon.java

```diff
@@ -1,2 +1,11 @@
 public class GrassPokemon extends Pokemon {
+    private String grassthrower = "grassthrower";
+
+    public void grassthrower() {
+        System.out.println(grassthrower);
```

Comment:
Hier zou je voor de leuk heel goed een string concatenation ( string + string ) kunnen toepassen, bijvoorbeeld (this.getName() + " attacks with " + grassthrower)